wizard/WizardHome.qml

```diff
@@ -196,10 +196,13 @@ Rectangle {
                         var item = dataModel.get(currentIndex).nettype.toLowerCase();
                         if(item === "mainnet") {
                             persistentSettings.nettype = NetworkType.MAINNET
+                            persistentSettings.wallet_path = ""
                         } else if(item === "stagenet"){
                             persistentSettings.nettype = NetworkType.STAGENET
+                            persistentSettings.wallet_path = ""
                         } else if(item === "testnet"){
                             persistentSettings.nettype = NetworkType.TESTNET
+                            persistentSettings.wallet_path = ""
```

Comment:
If there's no 4th network type, then the `wallet_path` might as well be reset once for all 3 types between lines 206 & 207

Reply:
@mj-xmr thanks for reviewing! I have updated, re-tested and squashed